src/Angor/Client/Storage/ClientStorage.cs

```diff
@@ -93,6 +93,11 @@ public void AddFounderProject(params FounderProject[] projects)
 
         _storage.SetItem("founder-projects", ret.OrderBy(_ => _.ProjectIndex));
     }
+    
+    public void ClearFounderProjects()
```

Comment:
Do we need this when we have DeleteFounderProjects?

Reply:
removed